### ISSUE TITLE
Implement default announce implementation in Fluent UI

### ISSUE THREAD

**Issue by ling1726** [microsoft/fluentui#29371]
https://github.com/microsoft/fluentui/pull/28654 added a shared context to create an announce contract in Fluent UI. Components like Toast and MessageBar would benefit from being able to leverage context managed aria-live narration but the library does not have a concrete implementation for an announcer.

This issue trackes the:
* Implementation of a default announcer in Fluent UI
* Documentation of the announce context and announcer
